pkgs/development/libraries/nss/generic.nix

```diff
@@ -114,6 +121,11 @@ stdenv.mkDerivation rec {
         --enable-legacy-db \
         --target ${target} \
         -Dhost_arch=${host} \
+    ''
+    + lib.optionalString (stdenv.hostPlatform != stdenv.buildPlatform) ''
+      -DOS=${host_system} \
+    ''
+    + ''
         -Duse_system_zlib=1 \
         --enable-libpkix \
         -j $NIX_BUILD_CORES \
```

Comment:
This actually isn't a mass rebuild, but if we're going to `staging` I would like to make the list of argument in Nix and then splice in bash, rather than directly making in bash.